doc/manual/src/glossary.md

```diff
@@ -23,9 +30,10 @@
 
 - [instantiate]{#gloss-instantiate}, instantiation
 
-  Translate a [derivation] into a [store derivation].
+  Save an evaluated [derivation] as a low-level [store derivation] in the Nix store.
```

Comment:
Hm... since I'm still quite unsure about what a derivation at the language level *is*, I have a bit of a hard time qualifying a derivation as evaluated or not. We can argue about the `derivation` primop with arguments being evaluated, but that sentence was deliberately bare-bones to avoid opening a can of worms or introducing irrelevant detail.

I also doubt that qualifying a store derivation as "low-level" helps. I know that the first entry of this glossary and the `nix-build` manual page call store derivations "low-level", but that's just because I haven't gotten to it yet. They're only (relatively) low-level from the perspective of anything outside of the store, and still they're exposed and dealt with explicitly in all kinds of interactions. And in terms of sentence structure, adding it as an adjective can be read as that there are at least two types of store derivations, regular ones and "low-level" ones.

At the very least, let's avoid the "low-level":

```suggestion
  Save an evaluated [derivation] as a [store derivation] in the Nix store.
```

But I don't think it says something wrong or misleading as it was.

Reply:
It was a [suggested effort](https://github.com/NixOS/nix/pull/9378#discussion_r1398322002) to avoid the word "translate". I agree that the fuzziness of what exactly a derivation *is* makes this wording awkward. I don't mind reverting this part if you prefer.

Reply:
"Translate" is indeed not great. Produce x from y?